packages/expo-system-ui/src/SystemUI.ts

```diff
@@ -0,0 +1,288 @@
+ * * "unspecified": Default
+ */
+// export type Appearance = 'light' | 'dark' | 'auto' | 'default';
+
+/**
+ * Changes the Android Status Bar's background color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * SystemUI.setStatusBarBackgroundColor("white");
+ * ```
+ */
+export function setStatusBarBackgroundColor(color: ColorValue): Promise<void> {
+  assertIsOnPlatform('setStatusBarBackgroundColor', ['android']);
+  const colorNumber = processColor(color);
+  return ExpoSystemUI.setStatusBarBackgroundColor(colorNumber);
+}
+/**
+ * Gets the Android Status Bar's background color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * const color = await SystemUI.getStatusBarBackgroundColor();
+ * ```
+ */
+export function getStatusBarBackgroundColor(): Promise<ColorValue> {
+  assertIsOnPlatform('getStatusBarBackgroundColor', ['android']);
+  return ExpoSystemUI.getStatusBarBackgroundColor();
+}
+
+/**
+ * Changes the Android Navigation Bar's background color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * SystemUI.setNavigationBarBackgroundColor("white");
+ * ```
+ */
+export function setNavigationBarBackgroundColor(color: ColorValue): Promise<void> {
+  assertIsOnPlatform('setNavigationBarBackgroundColor', ['android']);
+  const colorNumber = processColor(color);
+  return ExpoSystemUI.setNavigationBarBackgroundColor(colorNumber);
+}
+/**
+ * Gets the Android Navigation Bar's background color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * const color = await SystemUI.getNavigationBarBackgroundColor();
+ * ```
+ */
+export function getNavigationBarBackgroundColor(): Promise<ColorValue> {
+  assertIsOnPlatform('getNavigationBarBackgroundColor', ['android']);
+  return ExpoSystemUI.getNavigationBarBackgroundColor();
+}
+
+/**
+ * Changes the Android Navigation Bar's Divider color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * SystemUI.setNavigationBarDividerColor("red");
+ * ```
+ */
+export function setNavigationBarDividerColor(color: ColorValue): Promise<void> {
+  assertIsOnPlatform('setNavigationBarDividerColor', ['android']);
+  const colorNumber = processColor(color);
+  return ExpoSystemUI.setNavigationBarDividerColor(colorNumber);
+}
+
+/**
+ * Gets the Android Navigation Bar's Divider color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * const color = await SystemUI.getNavigationBarDividerColor();
+ * ```
+ */
+export function getNavigationBarDividerColor(): Promise<ColorValue> {
+  assertIsOnPlatform('getNavigationBarDividerColor', ['android']);
+  return ExpoSystemUI.getNavigationBarDividerColor();
+}
+
+/**
+ * Changes the Android Navigation Bar's visibility.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * SystemUI.setNavigationBarVisibility("hidden");
+ * ```
+ */
+export function setNavigationBarVisibility(visibility: 'visible' | 'hidden'): Promise<void> {
```

Comment:
Can you define `'visible' | 'hidden'` as a separate type (for example `NavigationBarVisibility`)?